### ISSUE TITLE
Padding in integration function sets currents to zero

### ISSUE THREAD

**Issue by kyralianaka** [jaxleyverse/jaxley#220]
In the integrate function, 

```
if t_max_steps > i_current.shape[0]:   
    i_current = jnp.zeros((t_max_steps, i_current.shape[1]))
```

sets the entire current input to just zeros if the length of the current input is less than the total integration time instead of padding as indicated in one of the comments. 

@michaeldeistler I can fix this quickly, but should the padding be done at the beginning or end of the current? Towards the beginning would allow model dynamics to stabilize, but then one might have to be more careful about aligning the results with other simulations.
